src/shared/sleep-config.c

```diff
@@ -196,8 +197,8 @@ static int read_battery_capacity_percentage(sd_device *dev) {
         return battery_capacity;
 }
 
-/* If battery percentage capacity is <= 5%, return success */
-int battery_is_low(void) {
+/* If a battery whose percentage capacity is <= 5% exists, and we're not on AC power, return success */
+int battery_is_present_and_low(void) {
```

Comment:
Hmm, but this new name doesn't reflect the new semantics. It is more precise, so one would expect that this extra precision describes the logic, but that's not the case. The old name was better…

Reply:
how so? it returns true if the battery is low, false otherwise

Reply:
Maybe something like `battery_is_discharging_and_low` then?